addons/settings/BW_Settings.inc.sqf

```diff
@@ -64,6 +66,7 @@ _settings = [
 [QACEGVAR(nightvision,aimDownSightsBlur), 0.25, false, true],
 [QACEGVAR(overheating,enabled), false],
 [QACEGVAR(refuel,rate), 10],
+[QACEGVAR(refuel,progressduration), 0.1],
```

Comment:
this might have been for testing, is this the time it takes to attach fuel line to vehicle?

Reply:
yeah, I think it was 0 on server,

Reply:
yea, should probably not be so low, but the default is slightly too long as well, can we make it like 2 seconds